CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+Version 2.0.3 *(2021-08-10)*
+----------------------------
+- Updated support for user survey data by adding `analyticsId` and making some other minor semantic changes to survey models.
```

Comment:
We should be more explicit about the changes and reference the field instead of the JSON key
```markdown
- Improved support for user survey data by adding an `analyticsId` property to `SurveyQuestion`, `SurveyResponseChoice`, and `UserSegmentSurvey`. This property replaces any `id` properties the class originally had.
- Renamed the `SurveyQuestion.emojiTitle` property to `titleEmoji`.
- Added a `resourceKey` property to `SurveyResponseChoice`.
```